.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Could you add `.vscode` to the .gitignore file. It should avoid adding these vscode config things then.